server/backend/testspg/jwt_utils.py

```diff
@@ -12,12 +12,18 @@
 COGNITO_USERNAME = f"test-idir_{IDP_USER_GUID}@idir"
 IDIR_USERNAME = "COGUSTAF"
 
-# for testing purpose,
-# we added this user as FOM DEV delegated admin in local sql,
-# to manage role FOM REVIEWER and FOM SUBMITTER with forest client number 00001018
+# Users setup (key identifier: cognito_user_id) for testing purpose.
+# We added these user "TEST-3-LOAD-CHILD-1"(BCEID) and "PTOLLEST"(IDIR) as
+#   FOM DEV delegated admin in local flyway sql, to manage role for
+#   FOM REVIEWER and FOM SUBMITTER with forest client number 00001018
 COGNITO_USERNAME_BCEID = (
```

Comment:
Hi Ian, sorry I just feel maybe we can also call them "COGNITO_USERNAME_BCEID_DELEGATED_ADMIN" and COGNITO_USERNAME_IDIR_DELEGATED_ADMIN"? so we know they are delegated admin

Reply:
I was not sure about it, 50/50, but no problem, I added now.